rss/serializers/atom_1_0.py

```diff
@@ -87,7 +84,8 @@ def get_xml(self: Serializer, eprints: EPrintSet) -> str:
             for author in eprint.authors:
                 author_list = {"name": author.full_name}
                 entry.author(author_list)
-                # TODO - How can arxiv-specific affiliation elements be added to authors?
+                if len(author.affiliations):
```

Comment:
This is technically fine, but consider an explicit comparison for clarity (``> 0``)

Reply:
Done, thanks!